react/features/recording/reducer.js

```diff
@@ -1,18 +1,32 @@
 import { ReducerRegistry } from '../base/redux';
+import { PersistenceRegistry } from '../base/storage';
 import {
     CLEAR_RECORDING_SESSIONS,
     RECORDING_SESSION_UPDATED,
-    SET_PENDING_RECORDING_NOTIFICATION_UID
+    SET_PENDING_RECORDING_NOTIFICATION_UID,
+    SET_STREAM_KEY
 } from './actionTypes';
 
 const DEFAULT_STATE = {
     sessionDatas: []
 };
 
+/**
+ * The name of the Redux store this feature stores its state in.
+ */
+const STORE_NAME = 'features/recording';
+
+/**
+ * Sets up the persistence of the feature {@code recording}.
+ */
+PersistenceRegistry.register(STORE_NAME, {
+    streamKey: true
+}, DEFAULT_STATE);
```

Comment:
A part of me is actually concerned about storing the key in local storage. If the user signs out of google, the key is still accessible. It concerns me but not that greatly for now. (The lack of a signout feature I also wonder about and that's on me.)

Reply:
I think you are right. Maybe we should only have it on mobile until we implement sign-in there. I'll change it

Reply:
Do you still plan to make this change?

Reply:
Yep, I did it